sdk/identity/identity/src/msal/nodeFlows/msalOpenBrowser.ts

```diff
@@ -52,7 +53,34 @@ export class MsalOpenBrowser extends MsalNode {
         claims: options?.claims,
         correlationId: options?.correlationId,
         loginHint: this.loginHint,
-      });
+      };
+      if (hasNativeBroker() && this.enableBroker) {
+        this.logger.verbose("Authentication will resume through the broker");
+        if (this.parentWindowHandle) {
+          interactiveRequest.windowHandle = Buffer.from(this.parentWindowHandle);
+        } else {
+          // error should have been thrown from within the constructor of Interactive Browser Credential
+          this.logger.warning(
+            "Parent window handle is not specified for the broker. This may cause unexpected behavior.Please provide the parentWindowHandle."
```

Comment:
```suggestion
            "Parent window handle is not specified for the broker. This may cause unexpected behavior. Please provide the parentWindowHandle."
```